roles/csi-driver/templates/driver/rbac-snapshotter.j2

```diff
@@ -0,0 +1,68 @@
+# Together with the RBAC file for external-provisioner, this YAML file
```

Comment:
I think this file can be removed, the csi driver doesn't support snapshotting (yet)